src/main/java/mate/academy/spring/validation/RepeatPassword.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = RepeatPasswordValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})

```

Comment:
to make it work you need to place it over the class

Reply:
are u sure? they are over interface, but in video i didn`t see the same for class

Reply:
![image](https://user-images.githubusercontent.com/118391929/222444132-b7e38074-3ff1-48c0-9d3e-462840e4e2d5.png)